BasiliskII/src/adb.cpp

```diff
@@ -429,9 +429,9 @@ void ADBInterrupt(void)
 			r.d[0] = (mouse_reg_3[0] << 4) | 0x0c;	// Talk 0
 			Execute68k(r.a[1], &r);
 
-			old_mouse_button[0] = mb[0];
-			old_mouse_button[1] = mb[1];
-			old_mouse_button[2] = mb[2];
+			old_mouse_button[0] = mb[0] != 0;
+			old_mouse_button[1] = mb[1] != 0;
+			old_mouse_button[2] = mb[2] != 0;
```

Comment:
Instead of doing this here and !! in the if, can you make local vars so you only do it once?